activities/notifications.go

```diff
@@ -59,7 +60,19 @@ func (ns notificationServices) SendEmail(email string, message notifications.Tem
 	return err
 }
 
-func (ns notificationServices) SendTelegramMessage(chatID string, message notifications.Template) error {
+func (ns notificationServices) SendTelegramMessage(channelID string, message notifications.Template) error {
+	bot, err := tgbotapi.NewBotAPI(os.Getenv("TELEGRAM_BOT_TOKEN"))
+	if err != nil {
+		return err
+	}
+	switch t := message.(type) {
+	case notifications.SimpleNotification:
```

Comment:
I think the notifications templates stuff you made here was quite clean but here you go away from that.

Maybe extend Template to also have a RenderMarkdown() / RenderTelegramHTML() (or rename the other one to RenderEmailHtml() etc).
For markdown / html in telegram you need to set parse_mode, see https://github.com/go-telegram-bot-api/telegram-bot-api/blob/4126fa611266940425a9dfd37e0c92ba47881718/configs.go#L45C8-L45C8

Reply:
Updated